lola.html

```diff
@@ -0,0 +1,875 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>LOLA Portability for Activity Pub (0.2)</title>
+    <script
+      src="https://www.w3.org/Tools/respec/respec-w3c"
+      class="remove"
+      defer
+    ></script>
+    <script class="remove">
+      var respecConfig = {
+        specStatus: "unofficial",
+        isPreview: true,
```

Comment:
This makes a big banner at the bottom of the page. Happy leaving it there, but we'll need to remove it before moving forward.